keps/sig-auth/4940-psa-block-host-field-in-probes/README.md

```diff
@@ -112,6 +112,14 @@ probes with the Host field set when using the (about to be deprecated) API.
 This is implemented by [kubernetes PR 125271](https://github.com/kubernetes/kubernetes/pull/125271)
 that does exactly that.
 
+NOTE: Given there is still a use case where admins might be deploying the apiserver
```

Comment:
This needs to be more than just a "note"; the rest of the document still talks about it as though we're completely forbidding the field

Reply:
ah yea in my PR I am doing that.. but yes let me update the KEP throughout you are right..

Reply:
I think I got the important spots.. hopefully didn't miss any this time